lib/Test/MockFile.pm

```diff
@@ -156,6 +156,8 @@ Relative paths are not supported:
 
 =cut
 
+use constant DEFAULT_STRICT_MODE => 256;    # not explicitly set but enable
```

Comment:
Why `256`?

Reply:
whatever number positive above 1, I was considering `-1` at one point but would need some adjustment 
if you have a better suggestion as long as it s different than `1` it's fine
which probably means we need another constant for the `1` lol